Dockerfile

```diff
@@ -13,6 +13,11 @@ RUN apt-get update \
     && docker-php-ext-configure gd --with-png-dir=/usr --with-jpeg-dir=/usr \
     && docker-php-ext-install gd mysqli opcache
```

Comment:
Adding `&& rm -rf /var/lib/apt/lists/* ` at the end of this should remove a few bytes from the image